_data/banking.yml

```diff
@@ -841,6 +841,15 @@ websites:
       tfa: No
       lang: it
 
+    - name: President's Choice Financial
+      url: https://www.pcfinancial.ca
+      img: pcfinancial.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
+      doc: https://www.pcfinancial.ca/en/security
```

Comment:
The doc link doesn't describe how to use or setup 2fa on the site so it's better just to leave it out.
```suggestion
```